apps/infrastructure/src/modules/chatbot/dynamodb.tf

```diff
@@ -41,5 +41,19 @@ module "dynamodb_chatbot_sessions" {
       name = "userId"
       type = "S"
     },
+    {
+      name = "createdAt"
+      type = "S"
+    },
   ]
+
+  # GSI for query on created_at
+  global_secondary_index {
```

Comment:
I think we can use a local_secondary_index cause the hash key is the same. This way we also reduce cost